test/integration/api/access_tokens_test.rb

```diff
@@ -40,6 +40,7 @@ def setup
     assert_not_equal 'foobar', AccessToken.last!.value
   end
 
+
```

Comment:
:eyes:

Reply:
This was unaceptable: https://github.com/3scale/porta/pull/3939/commits/5411c45c3dbe717e724ceedb02d7797586c3b7db